_posts/developers/apps/inrupt-tutorial/2020-11-06-00_first-app.md

```diff
@@ -20,7 +22,7 @@ This tutorial creates an introductory application that uses Inrupt's
 [JavaScript Client Libraries](https://github.com/inrupt/solid-client-js) to read
 and write your name to your user profile.
 
-Note:  Although this tutorial uses Inrupt's JavaScript libraries, you can use other libraries (be it for JavaScript or other languages) that support Solid. For more information, see the [Solid Developer Tools & Libraries](https://solidproject.org/developers/tools/).
+Note: Although this tutorial uses Inrupt's JavaScript libraries, you can use other libraries (be it for JavaScript or other languages) that support Solid. For more information, see the [Solid Developer Tools & Libraries](https://solidproject.org/developers/tools/).
```

Comment:
```suggestion
Note: Although this tutorial uses Inrupt's JavaScript libraries, you can use other libraries (whether for JavaScript or other languages) that support Solid. For more information, see the [Solid Developer Tools & Libraries](https://solidproject.org/developers/tools/).
```

Reply:
Hey @TallTed  - thanks for this suggestion!  
Just as an FYI ... in this pull, this content will not show.  Instead, if people had an old link to this page, this will redirect to the inrupt's getting started tutorial (which the old page used to be based on). For a preview, see this link
https://deploy-preview-793--solidproject-dot-org.netlify.app/developers/tutorials/first-app

Reply:
Oh ... and I'm definitely open to committing the your suggestion because it is a good change.  But, since the content doesn't show ... not sure what the group prefers.

Reply:
I'm fairly confident that good changes are good changes, whether or not they're visible in a default view.

Also, as noted in https://github.com/solid/solidproject.org/pull/793#pullrequestreview-1536670732, the detail about Inrupt libraries *should* be visible.